daffodil-runtime1/src/main/scala/org/apache/daffodil/debugger/InteractiveDebugger.scala

```diff
@@ -435,9 +437,9 @@ class InteractiveDebugger(runner: InteractiveDebuggerRunner, eCompilers: Express
     }
   }
 
-  private def infosetToString(ie: InfosetElement): String = {
+  private def infosetToString(ie: InfosetElement, xmlOutputStyle: XMLOutputStyle): String = {
     val bos = new java.io.ByteArrayOutputStream()
-    val xml = new XMLTextInfosetOutputter(bos, true)
+    val xml = new XMLTextInfosetOutputter(bos, true, xmlOutputStyle)
```

Comment:
I'd suggest the debugger be opinionated about what kind of Infoset it outputs and just use the 2-arg constructor to get the default standard output style. I don't think it's important that this be configurable in he debugger. Passing aroudn this xmlOutputSyle in this and other functions can be removed.

And if it was, it should be controlled by a debugger setting, similar to other settings like breakOnFailure, removeHidden, infosetLines, etc. Tunables aren't really used to control the debugger.